test/math/FloatPoint3DTest.ooc

```diff
@@ -57,12 +58,81 @@ FloatPoint3DTest: class extends Fixture {
 //			FIXME: Equals interface
 //			expect(FloatSize2D parse(value), is equal to(this vector3))
 		})
```

Comment:
There are several other mentions of an equals interface, I will remove them as well.